src/Kernel-CodeModel/BlockClosure.class.st

```diff
@@ -384,6 +384,30 @@ BlockClosure >> method [
 	^ self compiledBlock
 ]
 
+{ #category : 'private timing' }
+BlockClosure >> microsecondClockValue [
+	"Answer the number of microseconds since Epoch"
+
+	^self primUTCMicrosecondsClock
+]
+
+{ #category : 'private timing' }
+BlockClosure >> microsecondsToRun [
```

Comment:
And maybe this method should belong to the same package as `BlockClosure>>bench` (I'd create maybe later a new package?)

Reply:
@Ducasse to me the answer to your comments above is here. I want timeToRun & co out of the kernel too. They should be packaged elsewhere, maybe together with Time in first instance.

Reply:
my goal was to have timeToRun in a Duration Package that is on top of Time's package.